api/scripts/team-acces/update-fwb-authentication-complements.js

```diff
@@ -0,0 +1,89 @@
+import url from 'url';
+
+import _ from 'lodash';
+import bluebird from 'bluebird';
+
+import { disconnect } from '../../db/knex-database-connection.js';
+import { checkCsvHeader, parseCsvWithHeader } from '../helpers/csvHelpers.js';
+import { usecases } from '../../lib/domain/usecases/index.js';
+import { FWB } from '../../lib/domain/constants/oidc-identity-providers.js';
+
+const MODULE_PATH = url.fileURLToPath(import.meta.url);
+const IS_LAUNCHED_FROM_CLI = process.argv[1] === MODULE_PATH;
+const TIMER_NAME = '';
+const REQUIRED_FIELD_NAMES = ['SUB', 'Firstname', 'Lastname', 'EmployeeNumber', 'Population'];
+
+async function main() {
+  const filePath = process.argv[2];
+  const parsingOptions = {
+    skipEmptyLines: true,
+    header: true,
+    transform: (value) => {
+      if (typeof value === 'string') {
+        value = value.trim();
+      }
+      if (_.isEmpty(value)) {
+        value = null;
+      }
+
+      return value;
+    },
+  };
+
+  await checkCsvHeader({
+    filePath,
+    requiredFieldNames: REQUIRED_FIELD_NAMES,
+  });
+
+  console.log('Reading and parsing csv data file... ');
+
+  const fwbAuthenticationComplementsData = await parseCsvWithHeader(filePath, parsingOptions);
+  const errors = [];
+
+  await bluebird.mapSeries(fwbAuthenticationComplementsData, async (fwbAuthenticationComplementData) => {
+    const {
+      SUB: externalIdentifier,
+      Firstname: given_name,
+      Lastname: family_name,
+      Population: population,
+      EmployeeNumber: employeeNumber,
```

Comment:
Si on modifie, il faudra aussi modifier cette partie

```suggestion
      sub: externalIdentifier,
      firstName: given_name,
      lastName: family_name,
      population,
      employeeNumber,
```

Reply:
Done ! :-)